webview/src/experiments/components/table/Table.test.tsx

```diff
@@ -380,19 +381,6 @@ describe('Table', () => {
         type: MessageFromWebviewType.RESIZE_COLUMN
       })
       mockedPostMessage.mockReset()
-
-      fireEvent.mouseDown(experimentColumnResizeHandle, {
-        bubbles: true
-      })
-
-      columnWidths.id = 353
```

Comment:
This line was mutating the state and after a discussion with @sroy3, we decided to go ahead and take off the second half of this test since the first half is most likely enough.